app/Genre.php

```diff
@@ -8,6 +8,15 @@ class Genre extends Model
 {
     protected $guarded = ['id'];
 
+    /**
+     * The attributes that should be cast to native types.
+     *
+     * @var array
+     */
+    protected $casts = [
+        'approved_at' => 'datetime',
```

Comment:
I don't know what is recommended. But I always stick dates in `protected $dates = ["approved_at"];` and use casts only if I need a different format.

Reply:
That was indeed my intention! Good catch.